draft/2020-07-28-this-week-in-rust.md

```diff
@@ -29,7 +29,7 @@ Thanks to [Zicklag](https://users.rust-lang.org/t/crate-of-the-week/2704/792) fo
 [submit_crate]: https://users.rust-lang.org/t/crate-of-the-week/2704
 
 # Call for Participation
-
+[Rust Hashmaps]:(https://frogtok.com/very-simple-guide-to-rust-hashmaps/)
```

Comment:
Did you by any chance mean to put this in the News & Blog Post section? It would make more sense under that heading (and ty for the great article!)

Reply:
yes I wanted to put it there